avaframe/com1DFA/com1DFA.py

```diff
@@ -361,7 +361,7 @@ def com1DFACore(cfg, avaDir, cuSimName, inputSimFiles, outDir, simHash=''):
     # set thickness values for the release area, entrainment and secondary release areas
     relName, inputSimLines, badName = prepareReleaseEntrainment(cfg, inputSimFiles['releaseScenario'], inputSimLines)
 
-    log.info('Perform %s simulation' % cuSimName)
+    log.debug('Perform %s simulation' % cuSimName)
```

Comment:
I think this is a really helpful comment in the log and while running the sim - an option to keep it on info level?

Reply:
For most applications, this is already printed via a log.info on com1DFACoreTask, which has, IMHO higher priority due to the parallel execution. If someone calls core directly, I think it is fair to say that they can drop to debug in case of problems...